.github/workflows/release-python-client.yaml

```diff
@@ -0,0 +1,57 @@
+name: Publish Python 🐍 distributions 📦 to PyPI and TestPyPI
+
+on:
+  workflow_dispatch: {}
+  schedule:
+    - cron: '0 3 * * *' # Nightly 10:00 pm EST; 3AM UTC
+
+jobs:
+  build-n-publish:
+    name: Build and publish Python 🐍 distributions 📦 to PyPI and TestPyPI
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+      - name: set semver
+        run: |
+          CURRENT_SEMVER=$(curl -s -X GET "https://jade.datarepo-dev.broadinstitute.org/configuration" -H "accept: application/json" | jq -r '.semVer|rtrimstr("-SNAPSHOT")')
+          echo "CURRENT_SEMVER=${CURRENT_SEMVER}" >> "$GITHUB_ENV"
+      - name: Use Node.js ${{matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14
+      - name: Install openapi-generator-cli
+        run: npm install @openapitools/openapi-generator-cli -g
+      - name: set version to 4.3.1
+        run: openapi-generator-cli version-manager set 4.3.1
+      - name: Generate Python client
+        run: |
+          openapi-generator-cli generate \
+          -i src/main/resources/api/data-repository-openapi.yaml \
+          -g python \
+          -o data-repo-client \
+          --additional-properties=projectName=data-repo-client,packageName=data_repo_client,packageVersion=${CURRENT_SEMVER} \
+          --skip-validate-spec
+      - name: Install pypa/build
+        working-directory: ./data-repo-client
+        run: >-
+          python -m pip install build --user
+      - name: Build a binary wheel and a source tarball
+        working-directory: ./data-repo-client
+        run: >-
+          python -m build --sdist --wheel --outdir dist/ .
+      - name: Publish distribution 📦 to Test PyPI
+        uses: broadinstitute/gh-action-pypi-publish@master
+        with:
+          password: ${{ secrets.TEST_PYPI_API_TOKEN }}
+          repository_url: https://test.pypi.org/legacy/
+          packages_dir: ./data-repo-client/dist
+          skip_existing: true
+#      - name: Publish distribution 📦 to PyPI
+#        if: startsWith(github.ref, 'refs/tags')
+#        uses: broadinstitute/gh-action-pypi-publish@master
+#        with:
+#          password: ${{ secrets.PYPI_API_TOKEN }}
```

Comment:
Just curious, how was this token created?

Reply:
Oh good question on the tokens, I'm also curious about this. Also, will the token expire?

Reply:
I made it through my account on pypi.org, afaik they don't expire